_includes/footer.html

```diff
@@ -8,7 +8,7 @@
 </div>
 
 <div class="row wrap">
-<a class="btn btn-primary pull-right" href="https://github.com/rancher/rancher.github.io/tree/master/{{page.path }}">Edit this page <i class="fa fa-pencil"></i></a>
+<a class="btn btn-primary pull-right" href="https://github.com/rancher/rancher.github.io/tree/master/{{page.path.replace("/latest/", "/" + page.version + "/") }}">Edit this page <i class="fa fa-pencil"></i></a>
```

Comment:
My mistake, I was thinking between the `{{` was javascript but I guess it's Jekyll.

I'm reading I might have to make a custom filter in ruby.